### PR TITLE
MOD Pizza spider.

### PR DESCRIPTION
**Pull request by youngpm**
I had to tweak the linked_data_parser as the geo tag was nested under the "location", not sure if this was a good idea though...  If there's a better way, let me know.

### CODE REVIEW

File: locations/linked_data_parser.py [hunk 1]
```diff
@@ -42,8 +42,12 @@ def find_linked_data(response, wanted_type) -> {}:
     def parse_ld(ld) -> GeojsonPointItem:
         item = GeojsonPointItem()
 
-        if geo := ld.get("geo"):
-            if isinstance(ld["geo"], list):
+        if (
```
Comment (Cj-Malone) [inline locations/linked_data_parser.py:45]: I'm not sure this is part of the spec, or a common problem. If not I'd prefer it moved into spider code. We'd have to stop using `LinkedDataParser.parse` instead use `LinkedDataParser.find_linked_data`, fix the result, then use `LinkedDataParser.parse_ld`, [eg](https://github.com/alltheplaces/alltheplaces/blob/ebca0409ede42f60c1c1f4a91a029817edeb9c89/locations/spiders/sunglass_hut.py#L18)

If it is common and we stick with this we need to fix `if isinstance(ld.get("geo"), list):` to `if isinstance(geo, list):`

File: locations/spiders/modpizza.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        item = LinkedDataParser.parse(response, "FastFoodRestaurant")
+
+        # The address isn't in the microdata.
+        item["street_address"] = response.css(
+            ".Address-field.Address-line1::text"
+        ).get()
+        item["city"] = response.css(".Address-city::text").get()
+        item["state"] = response.css(".Address-field.Address-region::text").get()
+        item["postcode"] = response.css(".Address-field.Address-postalCode::text").get()
+        item["country"] = "US"
```
Comment (Cj-Malone) [inline locations/spiders/modpizza.py:31]: For static values we put them in `item_attributes`, so `item_attributes = {"brand": "MOD Pizza", "brand_wikidata": "Q19903469", "country": "US"}`

File: locations/linked_data_parser.py (hunk 1 above)
Comment (youngpm) [reply locations/linked_data_parser.py:45]: @Cj-Malone I ran it through the [schema.org validator](https://validator.schema.org/#url=https%3A%2F%2Flocations.modpizza.com%2Fusa%2Fky%2Flexington%2F2217-war-admiral-way) and it reported no errors, so I am guessing it is valid?  

Happy to fix it up either way!

Comment (Cj-Malone) [reply locations/linked_data_parser.py:45]: OK, yeah you're right, it's to spec. [location](https://schema.org/location) can be a [Place](https://schema.org/Place) and that can contain geo.

So we just need to do:

```patch
-if isinstance(ld.get("geo"), list):
+if isinstance(geo), list):
```

**Review by Cj-Malone — APPROVED** [review]
LGTM, thank you.

**Pull request merged**